run/calling_dash.py

```diff
@@ -111,11 +112,38 @@ def main():
     all_predictions_df.loc[
         all_predictions_df["call_status"] == "Gebeld", "call_status"
     ] = "🟢"
-    all_predictions_df.loc[
-        all_predictions_df["call_status"] != "🟢", "call_status"
-    ] = "🔴"
+    all_predictions_df.loc[all_predictions_df["call_status"] != "🟢", "call_status"] = (
+        "🔴"
+    )
     pred_id = all_predictions_df.iat[st.session_state["pred_idx"], 0]
 
+    # load information related to call history
+    with Session() as session:
+        current_response = session.get(ApiPrediction, pred_id).callresponse_relation
+        current_patient_nmbr = session.get(
+            ApiPatient, patient_ids[st.session_state["name_idx"]]
+        )
+    if not current_response:
+        current_response = ApiCallResponse(
+            call_status="Niet gebeld",
+            call_outcome="Geen",
+            remarks="",
+            prediction_id=pred_id,
+        )
+    if not current_patient_nmbr:
```

Comment:
Not possible right? You always add a apiPatient entry in the api?

Reply:
You're right, this was necessary in an earlier version but can be removed